File: app/src/main/java/com/github/quentin7b/sn/ui/AboutActivity.java
package com.github.quentin7b.sn.ui;

import android.content.pm.PackageInfo;
import android.content.pm.PackageManager;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.AppCompatTextView;
import android.support.v7.widget.Toolbar;
import android.util.Log;

import com.github.quentin7b.sn.R;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;
import de.psdev.licensesdialog.LicensesDialog;
import de.psdev.licensesdialog.model.Notice;
import de.psdev.licensesdialog.model.Notices;

public class AboutActivity extends AppCompatActivity {

    @BindView(R.id.toolbar)
    Toolbar toolbar;

    @BindView(R.id.about_version_tv)
    AppCompatTextView versionTv;

    private LicensesDialog licensesDialog;


    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        ButterKnife.bind(this);

        setSupportActionBar(toolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        setTitle(R.string.action_about);

        PackageInfo pInfo = null;
        try {
            pInfo = getPackageManager().getPackageInfo(getPackageName(), 0);
            versionTv.setText(getString(R.string.about_version, pInfo.versionName));
        } catch (PackageManager.NameNotFoundException e) {
            Log.e("Aa", "Cant get package manager", e);
        }

        licensesDialog = new LicensesDialog.Builder(this)
                .setNotices(R.raw.notices)
                .setIncludeOwnLicense(true)
                .build();
    }

    @OnClick(R.id.about_licenses_tv)
    void onVersionClicked() {
        licensesDialog.showAppCompat();
    }
}

File: app/src/main/java/com/github/quentin7b/sn/ui/AboutActivity.kt
package com.github.quentin7b.sn.ui

import android.content.pm.PackageInfo
import android.content.pm.PackageManager
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.util.Log
import com.github.quentin7b.sn.R
import de.psdev.licensesdialog.LicensesDialog
import kotlinx.android.synthetic.main.activity_about.*

class AboutActivity : AppCompatActivity() {


    private var licensesDialog: LicensesDialog? = null


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)

        setSupportActionBar(toolbar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        setTitle(R.string.action_about)

        var pInfo: PackageInfo? = null
        try {
            pInfo = packageManager.getPackageInfo(packageName, 0)
            versionTV.text = getString(R.string.about_version, pInfo!!.versionName)
        } catch (e: PackageManager.NameNotFoundException) {
            Log.e("Aa", "Cant get package manager", e)
        }


        licensesDialog = LicensesDialog.Builder(this)
                .setNotices(R.raw.notices)
                .setIncludeOwnLicense(true)
                .build()

        versionTV.setOnClickListener { licensesDialog!!.showAppCompat() }
    }
}
